pkg/target/target.go

```diff
@@ -260,6 +271,7 @@ func (h *K8sValidationTarget) MatchSchema() apiextensions.JSONSchemaProps {
 				Type: "array",
 				Items: &apiextensions.JSONSchemaPropsOrArray{
 					Schema: &apiextensions.JSONSchemaProps{
+						Type: "object",
```

Comment:
can we uncomment the `matchLabels` part? k8s 1.10 is long deprecated at this point.

Reply:
I've uncommented it and added `type: object`